web/src/toga_web/libs.py

```diff
@@ -69,3 +60,17 @@ def create_element(
             element.appendChild(child)
 
     return element
+
+
+def add_event_listener(element_id, event_type, callback):
+    """
+    Utility method to attach event listener to an element by ID
+
+    :param element_id: The ID of the DOM element
+    :param event_type: The name of the event e.g. sl-change
+    :param callback: The function to be called when the event triggers
+    """
+
+    element = document.querySelector(f"#{element_id}")
+    if element:
```

Comment:
Are you sure this isn't masking other problems? Under what conditions will the element not exist? From my testing... this is returning `None` for *every* widget, presumably because the widget hasn't been added to the DOM immediately after creation.